### PR TITLE
common-arguments: add Hindi translation

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by suyashbhawsar** [comment]
@kbdharun Can you suggest a title to the PR?

**Comment by kbdharun** [comment]
Sure, I will update it.

**Comment by suyashbhawsar** [comment]
> Sure, I will update it.

Also, the changes should be reviewed by the native speakers, as Hindi is my secondary language.

**Comment by kbdharun** [comment]
@2KAbhishek @adityarjadhav, Can you review the changes?
Thanks in advance

> > Sure, I will update it.
> 
> Also, the changes should be reviewed by the native speakers, as Hindi is my secondary language.

**Comment by suyashbhawsar** [comment]
@shubham-kshetre

File: contributing-guides/translation-templates/common-arguments.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Only the left-alignment of the header gets lost and has to be re-added again (`|
 | es    | ruta/al/archivo      | ruta/al/directorio     | ruta/al/archivo_o_directorio      | paquete   |                   |
 | fa    |                      |                        |                                   |           |                   |
 | fr    | chemin/vers/fichier  | chemin/vers/dossier    | chemin/vers/fichier_ou_dossier    | paquet    | nom_d_utilisateur |
-| hi    |                      |                        |                                   |           |                   |
+| hi    | फ़ाइल/का/पथ            | निर्देशिका/का/पथ            | फ़ाइल_या_निर्देशिका/का/पथ                 | संकुल      | उपयोगकर्ता-नाम         |
```
Comment (kbdharun) [inline contributing-guides/translation-templates/common-arguments.md:19]:
```suggestion
| hi    | फ़ाइल/का/पथ            | निर्देशिका/का/पथ            | फ़ाइल_या_निर्देशिका/का/पथ                 | पैकेज      | उपयोगकर्ता-नाम         |
```
Would this (पैकेज) be applicable here?

> Note: Don't commit this, Let's hear what others think.

Comment (suyashbhawsar) [reply contributing-guides/translation-templates/common-arguments.md:19]: I've never heard anyone using the word `पैकेज` in Hindi. I got a Sanskrit word for it: `सम्पुटिततन्त्रांश`, which seems to make the things harder.

**Review by shubham-kshetre — APPROVED** [review]

**Review by 2KAbhishek — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. 

> Also, Thanks for the reviews ❤️

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**